books/dbt-tutorial/config.yaml

```
@@ -0,0 +1,9 @@
+title: "手を動かして学ぶ dbt + DuckDB" # 本のタイトル
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)